### PR TITLE
Mobile 1287 Move React Native deploy to Github Actions

### PR DESCRIPTION
**Pull request by khmMouna**
### What do these changes do?
Move React Native deploy to Github Actions.

### How did you verify these changes?
To be tested when we release the React Native plugin.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CI
+
+#on: [pull_request]
+
+on:
+  push:
+    branches:
+      - MOBILE-1285
```
[the remark this answers is not in the document]
Comment (khmMouna) [reply .github/workflows/ci.yml]: Yes you are alright. I added for test, I will remove it :) thank you

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - name: Set up Node.js
+        uses: actions/setup-node@master
```
[the remark this answers is not in the document]
Comment (khmMouna) [reply .github/workflows/release.yml]: Yes you are alright. I should use the same version. Thank you

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: CI
+
+#on: [pull_request]
```
[the remark this answers is not in the document]
Comment (khmMouna) [reply .github/workflows/ci.yml]: It's already done. I just need to rebase my branch.

File: .github/workflows/release.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (khmMouna) [reply .github/workflows/release.yml]: Yes, you are alright,I should use the same version. Thank you :)

[the remark this answers is not in the document]
Comment (BrianBatchelder) [reply .github/workflows/release.yml]: I'm still wondering why this job has two steps that use `actions/setup-node`:
```
       - uses: actions/setup-node@master
         with:
           node-version: 12
           registry-url: https://registry.npmjs.org/
       - name: Set up Node.js
         uses: actions/setup-node@master
```

[the remark this answers is not in the document]
Comment (khmMouna) [reply .github/workflows/release.yml]: I did not pay attention. I will fix it.

**Comment by rlepinski** [comment]
This needs to be rebased before we merge it but it currently looks fine to me.

**Pull request merged**